docs/docs/testing-your-assistant.mdx

```diff
@@ -377,6 +377,26 @@ The test script will also save a confusion matrix to a file called
 matrix shows how often the action was correctly predicted and how often an
 incorrect action was predicted instead.
 
+### Interpreting the generated warnings
+
+The test script will also generate a warnings file called `results/stories_with_warnings.yml`.
+This file contains all test stories for which [`action_unlikely_intent`](./default-actions.mdx#action_unlikely_intent)
+was predicted at any conversation turn but all actions from the original story were predicted correctly.
+However, if a test story originally consisted of `action_unlikely_intent` but wasn't predicted by
+the ensemble of policies, then the corresponding story will end up in `results/failed_test_stories.yml` as
+a failed story.
+
+The stories are sorted by the severity of `action_unlikely_intent`'s prediction.
+This severity is calculated by `IntentTEDPolicy` itself at prediction time.
+Higher the severity, the more unlikely is the intent and hence reviewing that particular
+conversation path becomes more critical.
+
+It should be noted that `action_unlikely_intent` is predicted by
+[`IntentTEDPolicy`](./policies.mdx#intenttedpolicy) which employs a machine learning based model
+under the hood and hence can result in multiple false warnings as well. Users are encouraged to go through the
+stories in the warnings file and verify if the conversation path is already present in one of the
+training stories and add it as a training story if it isn't.
+
```

Comment:
This is somewhat confusing to me, because it sounds like you are saying to copy your test stories into training stories.

Reply:
Hmm, that's a good point. What do you think about this - 

"It should be noted that `action_unlikely_intent` is predicted by
[`IntentTEDPolicy`](./policies.mdx#intenttedpolicy) which employs a machine learning based model
under the hood and hence can result in false warnings as well. You can choose to ignore such warnings if the conversation paths in these stories are already present in the training stories."

Reply:
Looks good!

Reply:
Still, is there an approach to get rid of all warnings? 

Can a test story be marked to ignore the warning?

If we do not give some utitity for that, you end up looking at the same warnings all the time, which is frustrating, and you might not spot new, real warnings.

Reply:
> is there an approach to get rid of all warnings?

You can run `rasa test` with `--no-warnings` flag.

> Can a test story be marked to ignore the warning?

Nope, we don't have functionality for that but seems like a very valid point. We could maybe reuse the `metadata` key of a story for users to explicitly mark it to be ignored for warnings. Something like - 

```
story: test x
metadata:
    warn: false
steps:
   - ...
```

I can't say for sure but it seems easy to do in `rasa test`. @TyDunn Do you think it makes sense to get this in the `2.8` release?
@alwx Can you please comment on the feasibility of this?

Reply:
We can definitely do that, and it doesn't seem hard so it's more like a question of how important it is.

Reply:
Thanks! @TyDunn It makes sense to me to do something like this. This wouldn't be release blocking though. Would you be fine pushing it post release as part of a prioritized bug fix (I know it isn't a bugfix really)?